data/json/npcs/factions.json

```
@@ -944,5 +944,85 @@
       { "power_min": 150, "id": "epilogue_faction_exodii_150" }
     ],
     "description": "A tribe of scavengers and survivors that have been through this before, and sacrificed what they must to make it through."
+  },
+  {
+    "type": "faction",
+    "id": "the_great_library",
+    "name": "The Great Library",
+    "likes_u": 30,
+    "respects_u": 30,
+    "known_by_u": false,
+    "size": 100,
+    "power": 100,
+    "food_supply": 115200,
+    "wealth": 25000000,
+    "currency": "FMCNote",
```

```suggestion
    "currency": "FlatCoin",
```
There's an unused currency item meant for the faction, so they should probably use it.